worlds/sc2/item_descriptions.py

```diff
@@ -654,6 +654,10 @@ def _ability_desc(unit_name_plural: str, ability_name: str, ability_description:
     item_names.CORRUPTOR_RESOURCE_EFFICIENCY: _get_resource_efficiency_desc(item_names.CORRUPTOR),
     item_names.PRIMAL_IGNITER_CONCENTRATED_FIRE: "Primal Igniters deal +15 damage vs light armor.",
     item_names.PRIMAL_IGNITER_PRIMAL_TENACITY: "Primal Igniters gain +100 health and +1 armor.",
+    item_names.TYRANNOZOR_TYRANTS_PROTECTION: "Tyrannozors grants nearby friendly units 2 armor.",

```

Comment:
Super minor (optional to change), but normally for buffs we write "+2" instead of "2" armor.

Reply:
Fixed (also fixed in another string I saw the issue in).